coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -59,6 +59,11 @@ interface ImageLoader {
      */
     fun clearMemory()
 
+    /**
+     * Remove a single item from memory cache.
+     */
+    fun clearMemory(key: String)
```

Comment:
Can we rename this to `invalidate`? The behaviour of this function is different enough from `clearMemory()` that it should probably have its own name. `clearMemory()` empties the bitmap pool as well as the memory cache whereas `clearMemory(key)` only removes a specific memory cache entry.